db/compaction_iterator.h

```diff
@@ -21,6 +21,24 @@
 
 namespace rocksdb {
 
+class SnapshotListFetchCallback {
+ public:
+  SnapshotListFetchCallback(uint64_t snap_refresh_nanos,
+                            size_t _key_skip = 1024)
+      : snap_refresh_nanos_(snap_refresh_nanos), key_skip_(_key_skip) {}
+  virtual void Refresh(std::vector<SequenceNumber>* snapshots,
+                       SequenceNumber max) = 0;
+  inline uint64_t snap_refresh_nanos() { return snap_refresh_nanos_; }
+  inline uint64_t key_skip() { return key_skip_; }
```

Comment:
nit: I believe inline declared functions are inlined even without the keyword.

Reply:
Inlining is pretty much on the liberty of the compiler. Here it meant more of a hint.